oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticFullTextIndexCommonTest.java

```diff
@@ -67,4 +72,27 @@ protected List<String> getExpectedLogMessage() {
         expectedLogList.add(log2);
         return expectedLogList;
     }
+
+    @Test
+    /*
+     * analyzers by name are not possible in lucene, this test can run on elastic only
+     */
+    public void fulltextSearchWithBuiltInAnalyzerName() throws Exception {
+        setup(singletonList("foo"), idx -> {
+            Tree anl = idx.addChild(FulltextIndexConstants.ANALYZERS).addChild(FulltextIndexConstants.ANL_DEFAULT);
+            anl.setProperty(FulltextIndexConstants.ANL_NAME, "german");
```

Comment:
How does Lucene handle if we specify an analyzer by name? Should we have a test for that error case?

Reply:
That's not supported in Lucene. Analyzers need to be fully qualified class names. Since this is possible in Elastic, I think it is right to have a test here only. I don't see the point of having the same test in Lucene that verifies that an Exception gets thrown.